Session/Open Groups/JoinOpenGroupVC.swift

```diff
@@ -44,7 +44,7 @@ final class JoinOpenGroupVC: BaseVC, UIPageViewControllerDataSource, UIPageViewC
     }()
 
     private lazy var scanQRCodeWrapperVC: ScanQRCodeWrapperVC = {
-        let result: ScanQRCodeWrapperVC = ScanQRCodeWrapperVC(message: "vc_join_public_chat_scan_qr_code_explanation".localized())
+        let result: ScanQRCodeWrapperVC = ScanQRCodeWrapperVC(message: nil)
```

Comment:
Just wanted to confirm this was intentionally removed? Might be good to also remove the phrase from the `.strings` files since I don't think it's used anywhere else

Reply:
This is intentionally removed as new the UX design has the QR code scanner entire screen and the description is removed. I'll remove the string as well.